### PR TITLE
docs: Youtube videos&playlists about Deep Learning

### PR DESCRIPTION
**Pull request by gokerguner**
## What type of PR is this? (check all applicable)

- [x] Added new link to category
- [ ] New category added
- [ ] Fixed broken link
- [ ] Typo fix
- [ ] Readme update
- [ ] Optimization
- [ ] Documentation update
- [ ] Frontend update

## Description
Beginner and Intermediate One-Piece Youtube Videos about Deep Learning.

### CODE REVIEW

**Comment by damla** [comment]
PR template üzerinde 2. başlık olan `description` alanını editleyerek doldurabilirseniz sevinirim 😊 @gokerguner

File: docs/artificial-intelligence/deep-learning.md [hunk 1]
```diff
@@ -5,7 +5,11 @@ sidebar_label: Deep Learning
 ---
 
 ## Beginner
+[Pytorch for Deep Learing - Full Course/ Tutorial](https://www.youtube.com/watch?v=GIsg-ZUy0MY)
+
```
Comment (damla) [inline docs/artificial-intelligence/deep-learning.md:9]:
```suggestion
```

File: docs/artificial-intelligence/deep-learning.md [hunk 2]
```diff
@@ -5,7 +5,11 @@ sidebar_label: Deep Learning
...
+[Applied Deep Learning With Pytorch - Full Course](https://www.youtube.com/watch?v=CNuI8OWsppg)
 
 ## Intermediate
```
Comment (damla) [inline docs/artificial-intelligence/deep-learning.md:12]:
```suggestion
## Intermediate

```

File: docs/artificial-intelligence/deep-learning.md [hunk 3]
```diff
@@ -5,7 +5,11 @@ sidebar_label: Deep Learning
 ---
 
 ## Beginner
+[Pytorch for Deep Learing - Full Course/ Tutorial](https://www.youtube.com/watch?v=GIsg-ZUy0MY)
```
Comment (damla) [inline docs/artificial-intelligence/deep-learning.md:8]:
```suggestion
## Beginner

[Pytorch for Deep Learing - Full Course/ Tutorial](https://www.youtube.com/watch?v=GIsg-ZUy0MY)
```

**Comment by gokerguner** [comment]
Düzenledim 🙏🏾 @damla

**Review by damla — APPROVED** [review]
Thanks for sharing @gokerguner  🥳🛳

**Pull request merged**